### ISSUE TITLE
Itemize what workshop participants will learn

### ISSUE THREAD

**Issue by agitter** [gitter-lab/ml-bio-workshop#47]
Related to #45, we want to explicitly specify what new skills workshop participants will take away from the workshop.  We can also list what skills participants will not learn in order to better define the scope and attract the right audience.  Once we have the desired skills, we can work backwards to redesign the workshop content and activities.

Let's start by listing some ideas and then select those we like:
- Can recognize and define common machine learning terms (classifier, hyperparameter, training data, etc.)
- Can reconstruct the elements of a machine learning pipeline from the Methods section of a scientific manuscript
- Can identify classifiers that are appropriate for different types of decision boundaries (e.g. linear versus non-linear)
- Can recognize valid and invalid ways to tune hyperparameters, for instance, those that do not indirectly train on the independent test set
- Can identify scenarios in which accuracy, ROC curves, or other metrics are misleading or not indicative of a useful classifier

Please add other ideas below.

**Comment by cmilica** [comment]
Should we close this issue?

**Comment by agitter** [comment]
Yes, I'm closing this
